homeassistant/components/netatmo/config_flow.py

```diff
@@ -66,6 +66,10 @@ def extra_authorize_data(self) -> dict:
     async def async_step_user(self, user_input=None):
         """Handle a flow start."""
         await self.async_set_unique_id(DOMAIN)
+
+        if self.hass.config_entries.async_entries(DOMAIN):
+            return self.async_abort(reason="already_setup")
```

Comment:
```suggestion
            return self.async_abort(reason="single_instance_allowed")
```

Should this be `single_instance_allowed` ?  (and update strings.json)